plugins/linstor.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: linstor
+spec:
+  version: v0.1.2
+  homepage: https://github.com/piraeusdatastore/kubectl-linstor
+  shortDescription: A plugin to control a LINSTOR cluster using kubectl
+  description: |
+    A plugin to execute LINSTOR commands via the kubectl command line.
```

Comment:
This doesn't communicate anything to the user. What is LINSTOR, what does the plugin do? Please add context/links. See other plugins.